charts/incubator/frigate/templates/_configmap.tpl

```diff
@@ -204,23 +263,34 @@ data:
         {{- if .Values.frigate.record.events.retain.render_config }}
         retain:
           default: {{ .Values.frigate.record.events.retain.default | default 10 }}
-          mode: {{ .Values.frigate.record.events.retain.mode | default "motion" }}
+          {{- with .Values.frigate.record.events.retain.mode }}
+          mode: {{ . }}
+          {{- end }}
           {{- with .Values.frigate.record.events.retain.objects }}
           objects:
           {{- range . }}
             {{ .object }}: {{ .days }}
           {{- end }}
           {{- end }}
         {{- end }}
+      {{- end }}
     {{- end }}
 
     {{- if .Values.frigate.snapshots.render_config }}
     snapshots:
       enabled: {{ ternary "True" "False" .Values.frigate.snapshots.enabled }}
+      {{- with .Values.frigate.snapshots.clean_copy }}
       clean_copy: {{ ternary "True" "False" .Values.frigate.snapshots.clean_copy }}
+      {{- end }}
+      {{- with .Values.frigate.snapshots.timestamp }}
       timestamp: {{ ternary "True" "False" .Values.frigate.snapshots.timestamp }}
+      {{- end }}
```

Comment:
You can't have `with` on booleans.. when you have `false` it will never render it.